components/31-molecules/form-steps/_form-steps.scss

```
@@ -1,6 +1,6 @@
 .form-steps,
 .wizard-steps { /// @deprecated
-  & ol {
+  .form-steps-list {
```

```suggestion
  ol.form-steps-list {
```
I would like to enforce developers to use 'ol'.